### PR TITLE
Remove shadow from disabled buttons.

### LINKED ISSUE #1866: material spec: button disabled style
Disabled raised buttons should not have any shadow.

Left - Google docs, right - MDL.
To fix this, open _button.scss file, scroll to "disabled" section and remove following lines:

```scss
@include shadow-2dp(); // This should be removed
```

------

Originally from #1864

### PR DESCRIPTION
**Pull request by Garbee**
Fixes #1866

### CODE REVIEW

File: src/_variables.scss [hunk 1]
```diff
@@ -132,14 +132,14 @@ $target-elements-directly: true !default;
...
-$button-hover-color: $button-primary-color !default;
+$button-hover-color: unquote("rgba(#{$color-black}, 0.20)") !default;
 $button-active-color: unquote("rgba(#{$palette-grey-500}, 0.40)") !default;
 $button-focus-color: unquote("rgba(#{$color-black}, 0.12)") !default;
 
 // Colored button colors.
 $button-primary-color-alt: unquote("rgb(#{$color-primary})") !default;
 $button-secondary-color-alt: unquote("rgb(#{$color-primary-contrast})") !default;
-$button-hover-color-alt: unquote("rgb(#{$color-primary})") !default;
+$button-hover-color-alt: unquote("rgba(#{$color-primary}, 0.20)") !default;
```
Comment (samccone) [inline src/_variables.scss]: unrelated changes?

File: src/button/_button.scss [hunk 2]
```diff
@@ -277,7 +287,6 @@ input.mdl-button[type="submit"] {
     &.mdl-button--disabled.mdl-button--disabled {
       background-color: $button-primary-color-disabled;
       color: $button-secondary-color-disabled;
-      @include shadow-2dp();
```
Comment (samccone) [inline src/button/_button.scss]: this PR should only have these changes right?

Comment (Garbee) [inline src/button/_button.scss]: Up, thanks for catching that. Accidentally pushed a diff commit up by not branching before starting new work. :( Easy enough fix!

**Comment by samccone** [comment]
:+1: LGTM

**Comment by samccone** [comment]
landing this good work @Garbee :)

**Comment by oddui** [comment]
Might also need to overwrite the shadow set on `.mdl-button--raised`?

**Comment by Garbee** [comment]
Good catch @oddui. Pushing a commit directly to address that one.

**Comment by Garbee** [comment]
Done via commit bcf7f7722409c77a69529a836213cd80de65e3e6

**Comment by samccone** [comment]
good catch @oddui :dancer:
